SPECS/systemd/systemd.signatures.json

```diff
@@ -5,10 +5,7 @@
   "10-oomd-per-slice-defaults.conf": "3f0d6882312affecddfdc5a204a44c34b90c573839f2a4850a26b44f528520bd",
   "10-timeout-abort.conf": "ae6b234f92bc22f1201a7572b59b454c9809f33c80d13f361b9674e1801acc37",
   "20-yama-ptrace.conf": "f7881466bff200865ec2c6b5f989ed35855ac45420a55ae5ace805f9e5111828",
-  "26494.patch": "1242a0db037a2c8e5de3c8b676aad63caaae60d674227ec58d1ea11f085a996b",
```

Comment:
Curious why these diffs are present

Reply:
These files either are not present, or are .patch files which are handled differently. I ran the tools with the auto hash update turned on so they got cleaned up. I can revert if desired.

Reply:
It is fine to keep. Thanks!